src/index.ts

```diff
@@ -73,7 +73,15 @@ const createDebugger = ({ resolveCyclic }: Configuration = defaultConfig) => (
         before,
         after,
       };
-      currentConnection.send('actionDispatched', state);
+
+      try {
+        currentConnection.send('actionDispatched', state);
+      } catch (error) {
+        state.action = { type: action.type };
+        state.before = { error };
+        state.after = { error };
+        currentConnection.send('actionDispatched', state);
+      }
```

Comment:
I found that the error information in flipper is converted into another form of object which is not informative. I suggest to have one more field for the error message.

```suggestion
      } catch (error) {
        state.action = { type: action.type };
        state.before = { error, message: error.toString() };
        state.after = { error, message: error.toString() };
        currentConnection.send('actionDispatched', state);
      }
```